dev-python/pylama/pylama-7.4.3.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Code audit tool for Python and JavaScript."
+
+HOMEPAGE="https://github.com/klen/pylama"
+SRC_URI="https://github.com/klen/pylama/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-3"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+python_compile() {
+	esetup.py build
+}
```

Comment:
Missing tests.

Reply:
And this one.

Reply:
I added this?

python_test() {
	esetup.py test
}

Reply:
Oh, I see it now. You've added it in wrong order, that's why I didn't notice.